subprojects/configuration-cache/src/main/kotlin/org/gradle/configurationcache/serialization/codecs/Codecs.kt

```diff
@@ -140,7 +138,7 @@ class Codecs(
 
             providersBlock()
 
-            bind(ListenerBroadcastCodec(listenerManager))
+            bind(ListenerBroadcastCodec)
```

Comment:
Would this be observable in anyway from a test? I would be keen in adding one, but would appreciate suggestions for scenarios that would cause a listener to be called (or not be called) when it should (or not)?

Reply:
Good point. It should be observable in a test that adds a listener via the `ListenerManager` from the `Project` scope for an event broadcasted by a task.